external-import/zerofox/src/collectors/mappers/c2DomainsToInfrastructure.py

```diff
@@ -17,6 +27,7 @@ def c2_domains_to_infrastructure(
 
     infrastructure = Infrastructure(
         name=f"{entry.domain}",
+        created_by_ref=created_by,
```

Comment:
The Author field (i.e. “created_by_ref”) must be filled with an ID. You pass the Identity object (which I imagine is the “ZeroFox” organization) complete, you only need to fill in the field with the ZeroFox Organization's ID.